cql3/statements/alter_keyspace_statement.cc

```diff
@@ -87,7 +87,8 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
                 const std::map<sstring, sstring>& current_rfs = ks.metadata()->strategy_options();
                 for (const auto& [new_dc, new_rf] : _attrs->get_replication_options()) {
                     auto it = current_rfs.find(new_dc);
-                    if (it != current_rfs.end() && !validate_rf_difference(it->second, new_rf)) {
+                    sstring old_rf = it != current_rfs.end() ? it->second : "0";
+                    if (!validate_rf_difference(old_rf, new_rf)) {
```

Comment:
The patch title: `cql: ALTER tablets KS cannot change RF by more than 1 in new DCs` doesn't seem to describe the change in this patch.

Reply:
That holds true for the PR title as well.

Reply:
Would that be better?

    cql: sum of abs RFs diffs cannot exceed 1 in ALTER tablets KS
    
    Tablets load balancer is unable to process more than a single pending
    replica, thus ALTER tablets KS cannot accept an ALTER statement which
    would result in creating 2+ pending replicas, hence it has to validate
    if the sum of absoulte differences of RFs specified in the statement is
    not greter than 1.

This is the title and description from the latest commit in this patch series and I think it describes the changes in a more precise manner. On the other hand maybe it's too precise, I didn't want to give that level of details, especially the formula to calculate RFs in the commit's title may seem like too much.

Reply:
I just realized this comment is attached to a specific commit. I reworded the titles. Let me know if it's any better now.